Automattic/vip-go-mu-plugins
# Query Monitor: Revise Object Cache to play with get_stats()

## Description
This only works once https://github.com/Automattic/wp-memcached/pull/127 has been deployed. Otherwise, it will fall back onto the old method if `get_stats()` doesn't exist. 
<img width="1564" alt="Screenshot 2023-01-30 at 10 19 08 AM" src="https://user-images.githubusercontent.com/16962021/215548214-a0a33721-0f2f-42ec-ac36-c75f17b8d7d7.png">


This PR plays nice with dark mode and adds a "Operations" sub-panel which allows for filtering and sorting of Memcache operations. 
<img width="342" alt="Screenshot 2023-01-31 at 11 08 51 AM" src="https://user-images.githubusercontent.com/16962021/215846402-b37fcaed-b86c-4534-92fa-0d9d3de66e32.png">



The main panel will display a summary of the statistics.

## Changelog Description

### Plugin Updated: Query Monitor

Re-factored Object Cache panel to play nice with dark mode and sort/filter operations

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Add https://github.com/Automattic/wp-memcached/pull/127 as drop-in
2) Go to Query Monitor > Object Cache and Query Monitor > Object Cache > Operations to see it matches screenshot. 
3) In Operations sub-panel, play around with sorting and filtering.
4) Revert step 1) to drop-in and expect it to still work as it did before
